packages/vm/README.md

```diff
@@ -9,14 +9,19 @@
 | Execution Context for the Ethereum EVM Implementation. |
 | ------------------------------------------------------ |
 
-This package provides an Ethereum `mainnet` compatible execution context for the
+Ethereum `mainnet` compatible execution context for
```

Comment:
Is it important to say `mainnet` here?  That seems to be too narrowly focused since this would work in any chain context that you can specify in our `common` chain setup, right?